src/environment.rs

```diff
@@ -128,6 +176,15 @@ pub async fn update_prefix(
         )
         .await?;
     }
+
+    create_prefix_location_file(
+        &prefix
+            .root()
+            .parent()
+            .map(|p| p.join(consts::PREFIX_FILE_NAME))
+            .ok_or_else(|| miette::miette!("We should be able to create a prefix file name."))?,
+    )
+    .with_context(|| "Failed to create prefix location file.".to_string())?;
```

Comment:
Please use lowercase for errors, that makes it cleaner when concatenating errors. E.g.:

```
Error: failed to create prefix location file.
```

is nicer than:

```
Error: Failed to create prefix location file.
```

Reply:
```suggestion
    .with_context(|| "failed to create prefix location file.".to_string())?;
```